mappings/net/minecraft/sortme/EightHorizontalDirection.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS ez net/minecraft/sortme/EightHorizontalDirection
```

Comment:
EightHorizontalDirection doesn't seem right, but im struggling  to come up with a better name.

Looking at wikipedia OrdinalDirection might be correct, but id like to hear what others say.

Reply:
It appears to record the loading status of adjacent chunks in chunk loading.

Reply:
> EightHorizontalDirection doesn't seem right, but im struggling to come up with a better name.
> 
> Looking at wikipedia OrdinalDirection might be correct, but id like to hear what others say.

CompassPoint could be an appropriate generic name for the class. Ordinal implies there are no cardinal directions in the enumeration.